cli/argconfig.c

```diff
@@ -470,6 +470,23 @@ static int cfg_long_suffix_handler(const char *optarg, void *value_addr,
 	return 0;
 }
 
+static int cfg_long_long_handler(const char *optarg, void *value_addr,
```

Comment:
Not sure why these unrelated commits showed up -- I'll resolve this.

Reply:
I'm cancelling this PR. Please use this one instead: https://github.com/Microsemi/switchtec-user/pull/226